### PR TITLE
NAS-107753 / 11.3 / Prevent LDAP queries for local user info

### PR DESCRIPTION
**Pull request by anodos325**
Also prevent lookups for UIDs < 1000 from LDAP (with ability to override
via LDAP auxiliary parameter).

### CODE REVIEW

File: src/middlewared/middlewared/etc_files/local/nslcd.conf [hunk 1]
```diff
@@ -18,6 +20,14 @@
             ldap = None
 
         ldap_enabled = ldap['enable']
+        for param in ldap['auxiliary_parameters'].splitlines():
+            if not param.startswith('nss_min_uid'):
```
Comment (themylogin) [inline src/middlewared/middlewared/etc_files/local/nslcd.conf:24]: Can it start with whitespace?

**Comment by kmoore134** [comment]
retest this please

**Review by themylogin — APPROVED** [review]

**Comment by william-gr** [comment]
We will try to have this only for 12.0+

**Comment by william-gr** [comment]
backport

**Pull request closed without merging**